### PR TITLE
Fix variable names due to latest changes of ansible-lint

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Fix variable names due to latest changes of ansible-lint

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by gravesm — APPROVED** [review]

**Comment by GomathiselviS** [comment]
@NilashishC @justjais Since this PR touches the roles that are responsible for the generation of network modules, can you please review the changes?

File: roles/scaffold_plugins/tasks/template.yml [hunk 1]
```diff
@@ -8,28 +8,28 @@
   block:
     - name: Set path to main plugin file for generic plugins
       ansible.builtin.set_fact:
-        path_to_plugin: "{{ parent_directory }}/{{ plugin['type'] }}/{{ plugin['name'] }}.py"
+        scaffold_plugins__path_to_plugin: "{{ parent_directory }}/{{ plugin['type'] }}/{{ plugin['name'] }}.py"
       when: plugin['type'] in generic_plugins
```
Comment (NilashishC) [inline roles/scaffold_plugins/tasks/template.yml:12]:
```suggestion
      when: plugin['type'] in scaffold_plugins__generic_plugins
```

File: roles/scaffold_plugins/tasks/template.yml [hunk 2]
```diff
@@ -8,28 +8,28 @@
...
 
     - name: Set path to main plugin file for specialized plugins
       ansible.builtin.set_fact:
-        path_to_plugin: "{{ parent_directory }}/modules/{{ plugin['name'] }}.py"
+        scaffold_plugins__path_to_plugin: "{{ parent_directory }}/modules/{{ plugin['name'] }}.py"
       when: plugin['type'] not in generic_plugins
```
Comment (NilashishC) [inline roles/scaffold_plugins/tasks/template.yml:17]:
```suggestion
      when: plugin['type'] not in scaffold_plugins__generic_plugins
```

File: roles/scaffold_plugins/tasks/template.yml [hunk 3]
```diff
@@ -71,7 +71,7 @@
   when: plugin['type'] in generic_plugins
```
Comment (NilashishC) [inline roles/scaffold_plugins/tasks/template.yml:71]:
```suggestion
  when: plugin['type'] in scaffold_plugins__generic_plugins
```

**Review by NilashishC — COMMENTED** [review]
There are a bunch of places where the variables have been renamed where they are being set, but not updated where they are being used. This breaks network module and generic plugin scaffolding. 

I'll try to fix them one by one and push the updates to this PR.

**Comment by alinabuzachis** [comment]
> There are a bunch of places where the variables have been renamed where they are being set, but not updated where they are being used. This breaks network module and generic plugin scaffolding.
> 
> I'll try to fix them one by one and push the updates to this PR.

That would be awesome! Thank you!

**Comment by NilashishC** [comment]
@alinabuzachis @GomathiselviS All green now! 🥳

**Review by NilashishC — APPROVED** [review]

**Pull request merged**